File: Python/quadradoAVA.py
# Classe Quadrado: Crie uma classe que modele um quadrado:
# Atributos: Tamanho do lado
# Métodos: Mudar valor do Lado, Retornar valor do Lado e calcular Área;

class quadrado :
    def __init__(self):
       self.__lado = 50
    def setdaLado (self, lado) :
        self.__lado = lado
    def gettraLado (self):
        return print(f'o tamanho dos lados sao de {self.__lado}')
    def calcArea (self) :
        print(f'o calculo da area do quadrado é{self.__lado * self.__lado}')

File: PHP/AVA1/quadradoAVA.php
<!-- Classe Quadrado: Crie uma classe que modele um quadrado:

Atributos: Tamanho do lado
Métodos: Mudar valor do Lado, Retornar valor do Lado e calcular Área; -->
<?php
    class Quadrado{
        private $lados;
        public function setLado ($lado){
          echo $this -> lados = $lado ;
       }
       public function getLado(){
           return $this -> lado;
       }
       public function calcArea(){
           $area = $this -> lado * $this -> lado ;
           echo "a Area do quadrado é "+ $area;
       }
    }
?>
